Newsletters/Yearn Finance Newsletter #46/Yearn Finance Newsletter #46.md

```diff
@@ -1,80 +1,80 @@
 
-You can view this report [here](https://github.com/yearn/yearn-pm/blob/master/financials/reports/2021Q2-yearn-quarterly-report.pdf).
+Puedes ver este reporte [Aquí](https://github.com/yearn/yearn-pm/blob/master/financials/reports/2021Q2-yearn-quarterly-report.pdf).
 
-## **Cumulative Protocol Revenue Reaches $41M**
+## **Los ingresos acumulados del protocolo alcanzan los 41M**
 
 ![](image4.png)
 
-Yearn has earned nearly $41 million in revenue, since launch (excluding the interest earned by depositors). In the past 30 days, nearly $6 million dollars in revenue has been generated by the protocol. An extrapolation of this data projects annual revenue to be nearly $60 million.
+Yearn ha generado ingresos de casi 41 millones de dolares desde su lanzamiento (excluyendo los intereses generados por los depositantes). En los últimos 30 días, casi 6 millones de dolares han sido generados en ingresos del protocolo. Esta data permite proyectar ingresos anuales de casi 60 millones de dolares.
 
-Yearn's cumulative revenue and other statistics are viewable [here](https://www.yfistats.com/).
+Los ingresos acumulados de Yearn y otras estadísticas son visibles [Aquí](https://www.yfistats.com/).
 
-## **Vaults At Yearn**
+## Las vaults de Yearn
 
-You can read a detailed description of the strategies for all of our active yVaults [here](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3).
+Puedes leer una descripción detallada de las estrategias para todas nuestras yVaults activas [Aquí](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3).
 
-## **Ecosystem News**
+## **Noticias del ecosistema**
 
-[Sign up to become an intern at the yAcademy DAO and audit live smart contracts](https://twitter.com/yAcademyDAO/status/1435866622556659717)
+[Registrate para convertirte en interno en el DAO de la yAcademy para auditar contratos inteligentes activos](https://twitter.com/yAcademyDAO/status/1435866622556659717) 
 
-[YFI/ETH SushiSwap liquidity tokens are available as collateral on OnX's Alpha Vaults](https://twitter.com/OnXFinance/status/1435229990681972741)
+[Los tokens de liquidez YFI/ETH de SushiSwap están disponibles como colateral en las vaults Alphade OnX](https://twitter.com/OnXFinance/status/1435229990681972741)
 
-[Yearn is supporting ETHGlobal and ETHOnline, with $4,000 in prizes available for hackers and the opportunity to collaborate with the Yearn team](https://twitter.com/iearnfinance/status/1436302183545196546)
+[Yearn está apoyando ETHGlobal y ETHOnline, con $4,000 en premios disponibles para hackers y la posibilidad de colaborar con el equipo de Yearn](https://twitter.com/iearnfinance/status/1436302183545196546)
```

Comment:
AS-IS: [Yearn está apoyando ETHGlobal y ETHOnline, con $4,000 en premios disponibles para hackers y la posibilidad de colaborar con el equipo de Yearn]


To-BE: [Yearn está apoyando ETHGlobal y ETHOnline, con 4000$ en premios disponibles para hackers y la posibilidad de colaborar con el equipo de Yearn]